hudi-kafka-connect/src/main/java/org/apache/hudi/connect/transaction/ControlEvent.java

```diff
@@ -108,7 +109,9 @@ public int getVersion() {
   @Override
   public String toString() {
     return String.format("%s %s %s %s %s %s", version, msgType.name(), commitTime,
-        Arrays.toString(senderPartition), coordinatorInfo.toString(), participantInfo.toString());
+        Arrays.toString(senderPartition),
+        (coordinatorInfo == null) ? "" : coordinatorInfo.toString(),
```

Comment:
if you simply do `cordinatorInfo` instead of `coordinatorInfo.toString()` you can avoid the NPE